snowplow.js

```diff
@@ -15,17 +15,19 @@ import Cookies from 'js-cookie'
 import { COOKIE_PREF_KEY, DOCS_SITE_URLS } from './src/constants/config'
 import { reloadOnce } from './src/helpers/reloadOnce'
 
-const setupBrowserTracker = () => {
+const createTrackerConfig = (cookieName) => {
   const appId = DOCS_SITE_URLS.includes(window.location.hostname)
     ? 'docs2'
     : 'test'
   const domain = location.host.split('.').reverse()
 
-  const trackerConfig = {
+  let trackerConfig = {
```

Comment:
why?

Reply:
I misunderstood what the const keyword means. I thought it meant that the object is a constant and can't be changed (which it is on line 41 below). But reading more about it I see that it only prevents re-assigning and redeclaration. Good to know!